pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -66,12 +67,13 @@
     "version": "2.14.1"
   },
   "aws": {
-    "owner": "terraform-providers",
+    "owner": "hashicorp",
     "provider-source-address": "registry.terraform.io/hashicorp/aws",
     "repo": "terraform-provider-aws",
-    "rev": "v2.65.0",
-    "sha256": "005vs1qd6payicxldc9lr4w6kzr58xw9b930j52g1q7hlddl5mbb",
-    "version": "2.65.0"
+    "rev": "v3.15.0",
+    "sha256": "0rxpdxg5p478sipbhq2x347gs5wrlwz4ggy9z007cbp34yhb2wka",
+    "vendorSha256": "0vapfnd4c8jb15pdjnnb97vgsvfakjvl1czccbfy0znhdk2ynz02",
```

Comment:
How would one update this? Shouldn't the update-provider script be updated to take care of that?

Reply:
Indeed, that is the intention, I have a PR that needs a little more work  https://github.com/NixOS/nixpkgs/pull/104696.

The aim is to hook it up to `updateScript`

Reply:
Allright then, I think this PR is good to go.